packages/common/Providers.tsx

```diff
@@ -7,7 +7,7 @@ export function ThemeProvider({ children, ...props }: ThemeProviderProps) {
   // @ts-ignore next-themes is old :/
 
   return (
-    <NextThemesProvider {...props} themes={['dark', 'light', 'deep-dark']}>
+    <NextThemesProvider {...props} themes={['dark', 'light']}>
```

Comment:
the `themes` prop overrides any themes passed in via props. Should the unstructured props be at the end?

Reply:
we don't use 'classic-dark' on anything but Studio, so we override it in studio